processing/src/main/java/org/apache/druid/query/scan/ScanResultValueTimestampComparator.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.scan;
+
+import com.google.common.primitives.Longs;
+import org.apache.druid.java.util.common.UOE;
+import org.apache.druid.segment.column.ColumnHolder;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+public class ScanResultValueTimestampComparator implements Comparator<ScanResultValue>
```

Comment:
Could you make a short description of how this one orders the values?

Reply:
Addressed